templates/cases/case_read.haml

```diff
@@ -151,9 +151,31 @@
                   [[ key ]]
                 : [[ val ]]
 
+        - for pod in pods
+            %div{
+              ng-controller:"={ pod.pod_type.controller }",
+              ng-init:'init && init(={ pod.config.index }, ={ case_id }, ={ pod.config_json })',
+              "={ pod.pod_type.directive }":"true"
+            }
```

Comment:
You can just add it as an html tag. The hamlpy library has a few limitations I'm going to look into at some point, so you'll notice that where a tag name contains a dash we have to fall back to html